### PR TITLE
Fixed peerDependencies specification to work with React 16.

### PR DESCRIPTION
**Pull request by chaddjohnson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,8 +31,8 @@
     "immutable": "^3.8.1"
   },
   "peerDependencies": {
-    "react": "0.14.x || 15.x.x",
-    "react-dom": "0.14.x || 15.x.x"
+    "react": ">=0.14.x",
+    "react-dom": ">=0.14.x"
```
Comment (benjy) [inline package.json]: Would probably be better to add || 16.x.x

**Comment by chaddjohnson** [comment]
A temporary solution is to simply use Yarn which seems to not fail on peerDependencies issues.

File: package.json (hunk 1 above)
Comment (ric9176) [reply package.json]: @benjy can this be merged now? we'd love to move over to react16 with react-rte in production :-)

**Comment by benjy** [comment]
Fixed by #258

**Pull request closed without merging**